english/advice_from_mother/advice_from_mother.md

```diff
@@ -3,4 +3,4 @@ such as yourself should avoid LSD. Knowing now that this is all a bad
 trip, you use the bonfire to melt all the marshmallows, ignoring the 
 giant pile of fluff in one corner, watching the swirl and glitter of 
 the toasting surface until you can walk out of the corner and back 
-into the bar.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.